app/views/drafts/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row row-content">
+  <section class="action">
+    <div class="cta">
+      <%#= link_to 'Publish Draft', publish_collection_draft_path(get_resource), method: :post, class: 'eui-btn--blue spinner' %>
+      <%= link_to 'Delete Draft', "#delete-draft-modal", class: 'display-modal' %>
+      <div id="delete-draft-modal" class="eui-modal-content">
+        <a href="#" class="modal-close float-r"><i class="fa fa-times"></i><span class="is-invisible">Close</span></a>
+        <p>Are you sure you want to delete this draft?</p>
```

Comment:
Same minor detail here, we need to specify the type of draft.  We can use `resource_name` in these cases of shared views.